queries/lua/highlights.scm

```diff
@@ -181,12 +181,14 @@
 ;; Others
 
 (comment) @comment
+(comment) @spell
 
 (hash_bang_line) @comment
 
 (number) @number
 
 (string) @string
+(string) @spell
```

Comment:
I never quite liked spellchecking within strings... But if the people wants that, then let's have it.

Reply:
Me neither, but I think we are both in the minority.

We could add an option for it.

Reply:
Options for queries are a general todo. (Part of the "upstream and generalize predicate handling" tracking issue, now defered to 0.9.)

I'd like a general solution for such optional captures (e.g., for the fancier conceals) first.

Until then, we should do this consistently all-or-nothing.

Reply:
Can someone approve this then?

Reply:
Sure. It's consistent with upstream highlights, which is the main concern for me.

Reply:
Also, not a fan of the spellchecking within strings, how can I override this query locally?